expression/builtin_encryption.go

```diff
@@ -711,6 +711,52 @@ func (b *builtinSHA2Sig) Clone() builtinFunc {
 	return newSig
 }
 
+type sm3FunctionClass struct {
+	baseFunctionClass
+}
+
+func (c *sm3FunctionClass) getFunction(ctx sessionctx.Context, args []Expression) (builtinFunc, error) {
+	if err := c.verifyArgs(args); err != nil {
+		return nil, err
+	}
+	bf, err := newBaseBuiltinFuncWithTp(ctx, c.funcName, args, types.ETString, types.ETString)
+	if err != nil {
+		return nil, err
+	}
+	charset, collate := ctx.GetSessionVars().GetCharsetInfo()
+	bf.tp.SetCharset(charset)
+	bf.tp.SetCollate(collate)
+	bf.tp.SetFlen(40)
+	sig := &builtinSM3Sig{bf}
+	//sig.setPbCode(tipb.ScalarFuncSig_SM3) // TODO
```

Comment:
In the tikv side?

Reply:
This line could be un-commented after https://github.com/pingcap/tipb/pull/271 is merged.